src/generators/minecraft-pig/MinecraftPigGenerator.res

```diff
@@ -77,7 +77,7 @@ let textures: array<Generator.textureDef> = [
     standardHeight: 32,
   },
   {
-    id: "Pig (Vanilla) (Original)",
+    id: "Pig (Vanilla) (Programmer Art)",
```

Comment:
Thanks for fixing this texture name. I'm not familiar with what programmer art is, so if it's not an official in-game texture then I'm happy to remove it.

Reply:
Programmer Art is the name given to the resource pack that ships with 1.14+ that contains the textures before the update.

Reply:
Thanks @RubenVerg 👍